.travis.yml

```
@@ -125,16 +125,16 @@ jobs:
       name: Subcommands on Kubernetes
       script: make test-subcommand
 
-    # Run the unit, sanity, markdown, coverage and linter tests
+    # Run the unit, sanity, markdown, and coverage tests
     - stage: test
-      name: Unit, Sanity, and Markdown Tests
+      name: Unit, Sanity, Markdown, Coverage and Lint Tests
```

Nit: Since the code coverage happens during unit tests and the linter is part of the sanity test, maybe we keep the original name:
```suggestion
      name: Unit, Sanity, and Markdown Tests
```

And the comment in line 128 should be removed or updated to line up with the name. I'm in favor of just removing it because the name field makes it unnecessary.

---

Done as suggested 👍